src/cli.js

```diff
@@ -39,6 +39,8 @@ export default function CLI(argv, cwd) {
     .help("h").alias("h", "help")
     .version().alias("v", "version")
     .wrap(cli.terminalWidth())
+    .strict()
+    .showHelpOnFail(false)
```

Comment:
We should provide a message to `showHelpOnFail()`:
```js
    .showHelpOnFail(false, "Pass --help to see all available commands and options")
```

The existing `demandCommand()` usage should also include a message, to account for cases when `lerna` is called without arguments.
```js
    .demandCommand(1, "A command is required")
```